src/GraphQL/Types/Collections/ManualGraphTypeMappingProvider.cs

```diff
@@ -15,6 +15,6 @@ public ManualGraphTypeMappingProvider(Type clrType, Type graphType)
         _isOutputType = graphType.IsOutputType();
     }
 
-    public Type? GetGraphTypeFromClrType(Type clrType, bool isInputType, Type? preferredGraphType)
+    public Type? GetGraphTypeFromClrType([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties | DynamicallyAccessedMemberTypes.PublicMethods)] Type clrType, bool isInputType, Type? preferredGraphType)
```

Comment:
This is incorrect. Simply having a CLR mapping to a graph type does not require all public methods/properties to be rooted.  Calls to `Field` will provide a static reference to the properties needed.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/3519#issuecomment-1412933180

How to deal with warning here?